k3s.yaml

```diff
@@ -118,15 +112,10 @@ subpackages:
 
             # Replace upstream images with cgr.dev equivalents only if they exist
             case $repo in
-              # docker.io/rancher/klipper-helm)
-              #   image=$(crane digest --full-ref cgr.dev/chainguard/klipper-helm:latest)
-              #   ;;
-              # docker.io/rancher/klipper-lb)
-              #   image=$(crane digest --full-ref cgr.dev/chainguard/klipper-lb:latest)
-              #   ;;
-              # docker.io/rancher/local-path-provisioner)
-              #   image=$(crane digest --full-ref cgr.dev/chainguard/local-path-provisioner:latest)
-              #   ;;
+              docker.io/rancher/klipper-helm|docker.io/rancher/klipper-lb|docker.io/rancher/local-path-provisioner|docker.io/rancher/mirrored-pause)
```

Comment:
I pulled this in from https://github.com/wolfi-dev/os/pull/3657, which I will land separately once this new packaging structure lands